classes/interaction.lua

```diff
@@ -110,7 +128,11 @@ function Interaction:destroy()
     if self.globalType and ( self.entity or self.netId) then
         utils.wipeCacheForEntityKey(self.globalType, self.entity or self.netId)
     end
-    
+
+    if self.sprite ~= nil then
```

Comment:
could opt for conditional chaining operator here too to specifically check for the fict existing before accessing

Reply:
Switched to chaining operator in commit [f1c7f93](https://github.com/Sleepless-Development/sleepless_interact/pull/7/commits/f1c7f9380bf5f94cd95dcd7aba6c1c55d00c2f44)